datafusion/core/tests/sqllogictests/test_files/arrow_typeof.slt

```diff
@@ -176,10 +176,10 @@ drop table foo
 
 statement ok
 create table foo as select
-  arrow_cast(100, 'Decimal128(3,2)') as col_d128
+  arrow_cast(100, 'Decimal128(5,2)') as col_d128
```

Comment:
This results from - https://github.com/apache/arrow-rs/pull/3996

Which now complains (correctly) that 100 cannot be represented in an decimal with scale 2 and precision 3

Reply:
cc @viirya